deepsnap/batch.py

```diff
@@ -25,7 +25,9 @@ def __init__(self, batch=None, **kwargs):
         self.__slices__ = None
 
     @staticmethod
-    def collate(follow_batch=[], transform=None, **kwargs):
```

Comment:
this pattern appears in a few places i think:
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Hi,

Thanks for proposing this. It seems that it is not a bug because we want the users to pass in a list for `follow_batch` and if they don't pass in the list we will make the `follow_batch` an empty list. We are not going to append elements to the  `follow_batch` as appears in the link.

Reply:
yea, def not a bug from what i can tell so it's minor. however, this pattern can lead to very tricky issues to debug and is generally advised against so wanted to point it out. no worries if you prefer it the way it is.